charts/taraxa-info-api/templates/configmap.yaml

```diff
@@ -1,9 +1,18 @@
 apiVersion: v1
```

Comment:
I think you formatted all of these yaml files or something.

`{ { - ` and `} }` won't work
`{{-` and `}}` will work